tpm2/test/src/kmyth-test.c

```diff
@@ -71,51 +72,51 @@ int main(int argc, char** argv)
     return CU_get_error(); 
   }
 
-	// Create and configure storage key tools test suite
-	CU_pSuite storage_key_tools_test_suite = NULL;
-	storage_key_tools_test_suite = CU_add_suite("Storage Key Tools Test Suite",
-                                               init_suite, clean_suite);
-	if (NULL == storage_key_tools_test_suite)
-	{
-		CU_cleanup_registry();
-		return CU_get_error();
-	}
-	if (storage_key_tools_add_tests(storage_key_tools_test_suite))
-	{
-		CU_cleanup_registry();
-		return CU_get_error();
-	}
-
+  // Create and configure storage key tools test suite
+  CU_pSuite storage_key_tools_test_suite = NULL;
+  storage_key_tools_test_suite = CU_add_suite("Storage Key Tools Test Suite",
+					      init_suite, clean_suite);
+  if (NULL == storage_key_tools_test_suite)
+    {
+      CU_cleanup_registry();
+      return CU_get_error();
+    }
+  if (storage_key_tools_add_tests(storage_key_tools_test_suite))
+    {
+      CU_cleanup_registry();
+      return CU_get_error();
+    }
+  
```

Comment:
In reviewing the changes for this file, I noticed that the additions use a style that is different than that used in the parts that were original. While I do not have a significant preference for one style over the other, we attempt to utilize the indent utility on the source code that is not in the test directory to keep our style consistent. In response to this, I submitted pull request #107 in an attempt to address this (and found out that we were not running indent on many of the non-test header files as well).

Reply:
I ran indent on the file which should have fixed the formatting (but didn't do the other test files since you already put in a pull request for those.)